osu.Game.Rulesets.Osu/Utils/VectorHelper.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Game.Rulesets.Osu.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Utils
+{
+    public static class VectorHelper
```

Comment:
`Vector2Util` is better for matching the whole class?

Reply:
I guess `VectorUtils` is better. `Vector2` seems overly specific.